packages/react/src/icons/IconArrowRedo.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
I was thinking should these generated files also follow prettier rules.

Reply:
Yes they should, it can be done with a custom template or updating the preset template, which would be better?

Reply:
Good question. I think it would be better if every built icon file would go through prettier. When the files have different elements and attributes, the line breaks for example might be in different places for each file.

Reply:
That would at least be quality control and prevent passing components that don't pass prettier. Will add this.

Reply:
Prettier is now included in the workflow 👍